src/server/templates/settings.html

```diff
@@ -2665,10 +2674,18 @@ <h2>{{ __('System') }}</h2>
 	</div>
 	<div class="panel-content">
 		<div class="control-set">
+			{% if is_raspberry_pi %}
 			<button data-mfp-src="#shutdown_confirm" class="btn-danger open-mfp-popup">{{ __('Shut Down') }}</button>
 			<button data-mfp-src="#reboot_confirm" class="btn-danger open-mfp-popup">{{ __('Reboot') }}</button>
+			{% endif %}
+			{% if Debug %}
+			<button data-mfp-src="#kill_server_confirm" class="btn-danger open-mfp-popup">{{ __('Kill Server') }}</button>
+			{% endif %}
 			<a href="/hardwarelog" class="debug button-like">{{ __('Server Log') }}</a>
 			<button id="download_logs">{{ __('Download Logs') }}</button>
+			{% if node_fw_updatable %}
+			<a href="/updatenodes" class="debug button-like">{{ __('Update Nodes') }}</a>
```

Comment:
The `debug` class isn't appropriate here.

Reply:
Again, copied from (what I think is) your code, for the 'hardwarelog' button.  I tend to follow your lead on UI stuff. :]

Reply:
We should address later, but no need to hold up the PR.